google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -276,10 +285,25 @@ def build_cpr_model(
             no_cache=no_cache,
         )
 
+        env = [
```

Comment:
These should be populated in the dockerfile, not in the container spec, so that users can deploy the container with just the image URI and no extra configs.

Reply:
Yes, should be. Done.